### ISSUE TITLE
Couple of questions

### ISSUE THREAD

**Issue by perrivdb** [clean-docker/Magento2#48]
Hi there, I have a couple of questions.

Changing the hostname. Is this possible, and if so, how do I do it? localhost doesn't work that great with chrome and some forms. 

cron job. I can't get this to run the correct way, the customer grid is never updated. Any thoughts?

One or more integrations have been reset because of a change to their xml configs. I have been getting this error since the start, and cant start the integration. Never seen this before, do I have to do something about it?

**Comment by rafaelstz** [comment]
What's the integrations issue?

## Hostname
You can change your hostname changing the **localhost** at line 10 on this file https://github.com/clean-docker/Magento2/blob/master/docker-compose.yml and on your **/etc/hosts** file directly on your machine, like:

```
127.0.0.1	localhost	dev.local
```

## Cronjob
You can run the command below to install your cronjob.

```
n98 cron:install
```

**Comment by perrivdb** [comment]
Thanks for the reply, what do you mean exactly by changing localhost in line 10? line 10 just says 'volumes'. Cant seem to figure this out using docker

**Comment by rafaelstz** [comment]
Sorry, I mean **line 16**, change the localhost for dev.local, for example.

**Comment by perrivdb** [comment]
Thanks for the reply, got it working :) 

Only thiung that keeps me from using this box now is the syncing. Docker-sync never picks up any changes I make, until I fully restart the box or my computer. 

I'm running OSX High Sierra, with an 8700k and 16gb ram. If the changes would be picked up immediately this would be my go-to dev environment, but unfortunately they arent. 

Did you experience this yourself?

**Comment by rafaelstz** [comment]
Yes, sometimes I need to run these commands below to restart the docker-sync, I use the command `docker-sync sync` to try run again, and `docker-sync log` to check if everything is ok.
To restart the Docker sync you can run:

```
./stop
docker-sync clean
./start
```

I'm developing a new image using NGINX instead of Apache as well, that image doesn't use docker-sync, so you won't have this issue as your Mac has a great configuration.

https://github.com/Imagination-Media/docker-magento2
